lib/imports/api/authority/server/authority-manager.js

```diff
@@ -58,47 +75,151 @@ export class AuthorityManager {
         self.receiveSteps({ docId, clientId, version, stepsJSON });
       },
 
-      // open a doc, creating an authority for it
+      // open a doc, setting up its authority
       'ProseMeteor.openDoc' ({ docId }) {
         check(docId, String);
-        self.openDoc({ docId });
+        return self.openDoc({ docId });
+      },
+
+      // creates an authority in memory
+      'ProseMeteor.createAuthority' ({ docId, groupId = null }) {
+        check(docId, String);
+        check(groupId, Match.Maybe(String));
+        self.createAuthority({ docId, groupId });
       }
     });
   }
 
   /**
-  * Open a ProseMirror document. Creates a new Authority to track the doc.
+  * Open a ProseMirror document. Instructs appropriate server to create a server in memory and returns the server's URL.
```

Comment:
I would try to think of some other name for Authority then server. This can get confusing. ex: in this comment "find server to create server on". Maybe im reading this wrong and comment should just be cleaned up.